### PR TITLE
1.12.0rc0

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by jamshale — DISMISSED** [review]
I think we could consider making this a 1.2.0rc0 release due to the anoncreds issuance and presentation work which is now actually tested by the current `cheqd` did method plugin that got merged yesterday. The first additional anoncreds did method. I'm not exactly sure how to mention this in the acapy release notes but it is a considerable milestone.

I'll approve it as a `rc` release but wanted to mention that if it got missed.

**Comment by swcurran** [comment]
Sounds good--let me change it. On the iOS GitHub app there doesn't seem to be a way to make a PR draft...

**Comment by swcurran** [comment]
Updated to 1.2.0rc0 -- @jamshale, check it out!

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,11 +94,15 @@ While there are no breaking changes in this release that might impact production
...
+  - 1.2.0rc0 [\#3420](https://github.com/openwallet-foundation/acapy/pull/3420) [swcurran](https://github.com/swcurran)
   - 1.1.1rc0 [\#3372](https://github.com/openwallet-foundation/acapy/pull/3372) [swcurran](https://github.com/swcurran)
 
 - Dependabot PRs
-  - [Link to list of Dependabot PRs in this release](https://github.com/openwallet-foundation/acapy/pulls?q=is%3Apr+is%3Amerged+merged%3A2024-10-15..2024-12-03+author%3Aapp%2Fdependabot+)
+  - [Link to list of Dependabot PRs in this release](https://github.com/openwallet-foundation/acapy/pulls?q=is%3Apr+is%3Amerged+merged%3A2024-10-15..2024-12-24+author%3Aapp%2Fdependabot+)
+
+## 1.1.1
 
+ACA-Py Release 1.1.1 was a release candidate for 1.200. A mistake in the release PR meant the 1.1.1rc0 was tagged published to PyPi as Release 1.1.1. Since that was not intended to be a final release, the release changelog for 1.2.0 includes the Pull Requests that would have been in 1.1.1.
```
Comment (jamshale) [inline CHANGELOG.md:105]: 1.200 seems wrong?

**Review by jamshale — COMMENTED** [review]
One small comment.

File: CHANGELOG.md (hunk 1 above)
Comment (swcurran) [reply CHANGELOG.md]: And in fact is wrong.  Fixed.

**Review by jamshale — APPROVED** [review]

**Pull request merged**